extensions/amp-kaltura-player/0.1/amp-kaltura-player.js

```diff
@@ -74,41 +76,24 @@ class AmpKaltura extends AMP.BaseElement {
     return this.loadPromise(iframe);
   }
 
-  /** @private */
-  buildImagePlaceholder_() {
-    const imgPlaceholder = new Image();
-
-    setStyles(imgPlaceholder, {
-      'object-fit': 'cover',
-      'visibility': 'hidden',
-    });
+  /** @override */
+  createPlaceholderCallback() {
+    const placeholder = this.win.document.createElement('amp-img');
     const width = this.element.getAttribute('width');
     const height = this.element.getAttribute('height');
-    const partnerid = user().assert(
-      this.element.getAttribute('data-partner'),
-      'The data-partner attribute is required for <amp-kaltura-player> %s',
-      this.element);
-    const entryid = this.element.getAttribute('data-entryid') || 'default';
-    let src = `https://cdnapisec.kaltura.com/p/${encodeURIComponent(partnerid)}/thumbnail/entry_id/${encodeURIComponent(entryid)}`;
+    let src = `https://cdnapisec.kaltura.com/p/${encodeURIComponent(this.partnerId_)}/thumbnail/entry_id/${encodeURIComponent(this.entryId_)}`;
     if (width) {
       src += `/width/${width}`;
     }
     if (height) {
       src += `/height/${height}`;
     }
-
-    imgPlaceholder.src = src;
-    imgPlaceholder.setAttribute('placeholder', '');
-    imgPlaceholder.setAttribute('referrerpolicy', 'origin');
-
-    this.applyFillContent(imgPlaceholder);
-    this.element.appendChild(imgPlaceholder);
-
-    this.loadPromise(imgPlaceholder).then(() => {
-      setStyles(imgPlaceholder, {
-        'visibility': '',
-      });
-    });
+    placeholder.setAttribute('noprerender', '');
```

Comment:
I am actually not 100%. Sure was it only if the request is sending or setting cookies during prerender? I'll review the document. 

In this PR u added noprerender to items that uses the same domain for loading the main content and prerender content.